.github/workflows/ci_tests.yml

```diff
@@ -24,15 +24,15 @@ jobs:
 
     - name: Run EMMO Check - battery
       run: |
-        emmocheck --verbose --check-imported --url-from-catalog \
+        emmocheck --verbose --url-from-catalog \
```

Comment:
What is the reason for not checking imported ontologies?

Reply:
Bacause it then check emmo+++ and any other imported ontologies. If there are errors in those (which are not in this repo) the workflow will fail, but this is not something that can be fioxed in this repo.

Reply:
Good point